util/git/git.go

```diff
@@ -37,6 +37,9 @@ func IsCommitSHA(sha string) bool {
 // NormalizeGitURL normalizes a git URL for lookup and storage
 func NormalizeGitURL(repo string) string {
 	// preprocess
+	if repo == "*" {
+		return repo
+	}
```

Comment:
Is it possible to remove this? I consider it an error if NormalizeGitURL is ever called with `*`